.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+<!-- Please provide a general summary of your changes in the title above.
+
+If the change is specific to a particular PHP version, prefix the commit summmary/PR title with "PHP #.#".
+Mentioning the touched sniff also helps in keeping the git history scannable.
+
+Example: "PHP 8.5 | NewFunctions: detect new error handler functions (RFC)"
+-->
+
+<!--
+It is STRONGLY recommended to use the below template as a basis for your commit messages.
+This will allow for a better `git blame` experience.
+
+A good commit message for a commit to PHPCompatibility would typically look something like this:
+```
+PHP #.# | SniffName: detect something specific
```

Comment:
Just a question: given that sniff names are very long, does PHPCompatibility prescribe a char limit on commit subject lines (typically 72)?